package.json

```diff
@@ -28,5 +49,6 @@
     "react-dom": "^15.3.0",
     "react-relay": "^0.9.2",
     "sync-request": "^3.0.1"
-  }
+  },
+  "license" : "MIT"
```

Comment:
I correct myself , you're wrong. That is not a valid expression SPDX [http://spdx.org/](url)
Please look at this site, where you will find licenses SPDX [https://spdx.org/licenses/index.html](url).